src/index.css

```diff
@@ -59,26 +81,94 @@ nav a {
 
 .inner-container {
   display: grid;
-  grid-template-rows: 1fr 1fr 1fr;
+  grid-template-columns: repeat(3, minmax(0, 1fr));
+  gap: 15px;
   border: 1px solid #dbdada;
-  padding: 5px 10px;
+  padding: 20px;
+  justify-content: center;
+}
+
+.box {
+  padding: 5px 15px;
+  align-items: center;
+}
+
+.grey {
+  color: #888;
+}
+
+.each-book {
+  display: flex;
```

Comment:
- [ ] As requested by the previous code reviewer, please make sure that you keep your code [DRY](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) by using a reusable CSS class. For instance, you have this CSS property `display: flex;` repeated many times in your CSS code and it would be better if you give a class name like `d-flex` to all the elements you are displaying them flexbox and you write a CSS like this :point_down: and remove all the respected flexbox to keep your code DRY
  
  ```css
  .d-flex {
    display: flex;
  }
  ```